tests/chez/chez029/run

```diff
@@ -0,0 +1,3 @@
+$1 --no-banner BitCasts.idr < input
```

Comment:
Tests should always have `--no-color --console-width 0` since they are too much dependent on the particular system where the tests are run.

Reply:
Thanks for the tip!